dodo.py

```diff
@@ -647,12 +647,13 @@ def _optimize_images():
                 cwd=str(P.ROOT),
             )
 
-        yield dict(
-            name="post:images",
-            doc="clean sphinx-jsonschema duplicate ids",
-            actions=[_optimize_images],
-            file_dep=[B.YARN_INTEGRITY],
-        )
+        if not C.RTD:
```

Comment:
Probably doesn't make sense to skip on RTD since the task is about optimizing svgs in the docs folder.